### PR TITLE
Initial commit, created Hello world project

### PR DESCRIPTION
**Pull request by Oplia512**
Please check it

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+### Intellij ###
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio and Webstorm
+# Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
+
+# User-specific stuff:
+.idea/*
+.classpath
+.project
+.settings/*
+target/*
```
Comment (agawda) [inline .gitignore:13]: FYI: it's possible to include maven in www.gitignore.io and it adds `target` automatically.

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,152 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.java_academy.Battleships</groupId>
+    <artifactId>Battleships</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <name>BATTLESHIPS</name>
+    <url>http://maven.apache.org</url>
```
Comment (agawda) [inline pom.xml]: This url shouldn't be here.

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-checkstyle-plugin</artifactId>
+                <version>${checkstyle.version}</version>
+            </plugin>
+
+        </plugins>
+    </reporting>
+
```
Comment (agawda) [inline pom.xml:160]: :+1:

File: readme.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
+# Battleships 2
+
+The purpose of this project is to practise DDD (Delete Driven Development). You can read more about rules int this [article](https://en.wikipedia.org/wiki/Battleship_(game))
```
Comment (agawda) [inline readme.md]: practice* in* :stuck_out_tongue: Fix typos please.

File: src/test/java/com/java_academy/AppTest.java [hunk 5]
```diff
@@ -0,0 +1,38 @@
+package com.java_academy;
+
+import junit.framework.Test;
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
```
Comment (agawda) [inline src/test/java/com/java_academy/AppTest.java]: :crying_cat_face: Fix that before Tomek gets here.

File: readme.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+Java 1.8
+Maven 3.0.1
+
+```
+
+
+## Running the tests
+
+```
+To execute: mvn exec:java -Dexec.mainClass="com.javaAcademy.Battleship2.App"
```
Comment (agawda) [inline readme.md]: This is not a test, you could have another section for running the app.

File: readme.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+2. To test: mvn test
+```
+
+
+## TODO List
```
Comment (agawda) [inline readme.md]: :+1:

File: pom.xml [hunk 8]
```diff
@@ -0,0 +1,152 @@
...
+    <prerequisites>
+        <maven>3.0.1</maven>
+    </prerequisites>
+
+    <properties>
+        <!-- JAVA & ENCODING-->
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <java.version>1.8</java.version>
+
+        <!-- DEPENDENCIES -->
```
Comment (agawda) [inline pom.xml:20]: :+1:

File: pom.xml [hunk 9]
```diff
@@ -0,0 +1,152 @@
...
+
+    <name>BATTLESHIPS</name>
+    <url>http://maven.apache.org</url>
+
+    <prerequisites>
+        <maven>3.0.1</maven>
+    </prerequisites>
+
+    <properties>
+        <!-- JAVA & ENCODING-->
```
Comment (agawda) [inline pom.xml]: :+1:

**Review by agawda — CHANGES_REQUESTED** [review]
Remove junit. :v

File: pom.xml (hunk 2 above)
Comment (Devas) [reply pom.xml]: :+1:

File: pom.xml (hunk 9 above)
Comment (Devas) [reply pom.xml]: Use properties

File: pom.xml (hunk 3 above)
Comment (Devas) [reply pom.xml:160]: :+1:

File: readme.md (hunk 6 above)
Comment (Devas) [reply readme.md]: :+1:

File: src/main/java/com/java_academy/App.java [hunk 10]
```diff
@@ -0,0 +1,13 @@
+package com.java_academy;
+
+/**
+ * Hello world!
+ *
+ */
+public class App
```
Comment (Devas) [inline src/main/java/com/java_academy/App.java]: Use auto formatting for whole code - once you have opening brace in the same line and next time you have it line below

File: src/test/java/com/java_academy/AppTest.java [hunk 11]
```diff
@@ -0,0 +1,7 @@
+package com.java_academy;
+
+/**
+ * Unit test for simple App.
+ */
+public class AppTest{
```
Comment (Devas) [inline src/test/java/com/java_academy/AppTest.java]: Use auto formatting for whole code - one time you have opening brace in the same line and next time you have it in the line below

File: src/test/java/com/java_academy/AppTest.java [hunk 12]
```diff
@@ -0,0 +1,7 @@
+package com.java_academy;
+
+/**
+ * Unit test for simple App.
```
Comment (Devas) [inline src/test/java/com/java_academy/AppTest.java]: @Test ?

**Review by Devas — CHANGES_REQUESTED** [review]
Use auto formatting for code and properties in POM

File: src/test/java/com/java_academy/AppTest.java (hunk 12 above)
Comment (Oplia512) [reply src/test/java/com/java_academy/AppTest.java]: Please use more understandable comments

**Pull request closed without merging**